### PR TITLE
more general pypi ci

### PR DESCRIPTION
**Pull request by irexyc**
use `actions/checkout@v3` repo instead of git clone

### CODE REVIEW

File: builder/manywheel/build_wheel.sh [hunk 1]
```diff
@@ -20,8 +18,7 @@ docker run --rm -it \
     --env PLAT_NAME="${PLAT_NAME}" \
     --env USERID="${USERID}" \
     --env GROUPID="${GROUPID}" \
-    --env GIT_BRANCH="${GIT_BRANCH}" \
-    --env GIT_REMOTE="${GIT_REMOTE}" \
+    --volume "$(pwd)/../../:/lmdeploy" \
```
Comment (RunningLeon) [inline builder/manywheel/build_wheel.sh:21]: is `lmdeploy_root=$(pwd)/../lmdeploy` when the following step runs?
https://github.com/InternLM/lmdeploy/blob/450757b2bdf3b124cfa14950e4f8ed6f6d15550c/.github/workflows/pypi.yml#L34

Comment (irexyc) [reply builder/manywheel/build_wheel.sh:21]: yes, `cd "$(dirname "$0")"` will move workdir to where the scripts exists. And the original command has this line `--volume "$(pwd)/entrypoint_build.sh:/entrypoint_build.sh" \`

Comment (RunningLeon) [reply builder/manywheel/build_wheel.sh:21]: sorry. didn't see line 13

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**